### ISSUE TITLE
Wrong formatting in "Scenario details" for steps with "*" annotation instead of given/when/then

### ISSUE THREAD

**Issue by 180505** [serenity-bdd/serenity-core#1639]
In case `*` is used instead of `given/when/then`, the step in report has `<ul>` tag instead of `<p>` what results in wrong formatting (there is no css for `.scenario-docs ul` element). 
Using star annotation makes the scenario more readable, please check details: https://relishapp.com/cucumber/cucumber/docs/gherkin/using-star-notation-instead-of-given-when-then 

Best Regards,
Kasia

**Comment by wakaleo** [comment]
Good point; could you propose a PR with the correct CSS?

**Comment by 180505** [comment]
Hi, I cannot create new branches (remote: Permission to serenity-bdd/serenity-core.git denied to 180505), therefore I would not be able to create PR. But my proposal would be to just add following to `serenity-report-resources/src/main/resources/report-resources/css/core.css`:

```
.scenario-docs ul {
    padding-left: 3.5em;
    margin: 0px;
}

```

**Comment by wakaleo** [comment]
Fixed in 2.0.42
